creme/creme_core/tests/models/test_history.py

```diff
@@ -335,7 +336,8 @@ def test_edition_fk(self):
         img = FakeImage.objects.create(user=user, name='Grumpy Hayao')
 
         hayao.image = img
-        hayao.save()
+        with self.assertNumQueries(6):
```

Comment:
Could you add a comment with give the detail of the queries please (see creme.activities.tests.test_bricks.ActivityBricksTestCase.test_participants_brick), so when the assertion will fail, we can easily identify what has changed.

Reply:
ok nice, I will add the description in a comment.

Do you understand the two `SELECT` requests 5 and 6 ? 
I don't understand why update a contact image will make a `SELECT` in the `Alert` model :thinking: 

```
1. UPDATE `creme_core_cremeentity` SET `created` = '2024-01-31 15:25:31.186292', `modified` = '2024-01-31 15:25:31.233056', `entity_type_id` = 73, `header_filter_search_field` = 'Hayao Miyazaki', `is_deleted` = 0, `user_id` = 82, `description` = '', `uuid` = '3f56f7df37514c4890ae44d9d3336020', `sandbox_id` = NULL, `extra_data` = '{}' WHERE `creme_core_cremeentity`.`id` = 1258
2. UPDATE `creme_core_fakecontact` SET `civility_id` = NULL, `last_name` = 'Miyazaki', `first_name` = 'Hayao', `is_a_nerd` = 0, `loves_comics` = NULL, `phone` = NULL, `mobile` = '', `email` = '', `url_site` = '', `position_id` = NULL, `sector_id` = NULL, `address_id` = NULL, `is_user_id` = NULL, `birthday` = NULL, `image_id` = 1259 WHERE `creme_core_fakecontact`.`cremeentity_ptr_id` = 1258
3. SELECT `creme_core_cremeuser`.`id`, `creme_core_cremeuser`.`password`, `creme_core_cremeuser`.`last_login`, `creme_core_cremeuser`.`username`, `creme_core_cremeuser`.`last_name`, `creme_core_cremeuser`.`first_name`, `creme_core_cremeuser`.`email`, `creme_core_cremeuser`.`displayed_name`, `creme_core_cremeuser`.`date_joined`, `creme_core_cremeuser`.`is_active`, `creme_core_cremeuser`.`is_staff`, `creme_core_cremeuser`.`is_superuser`, `creme_core_cremeuser`.`role_id`, `creme_core_cremeuser`.`is_team`, `creme_core_cremeuser`.`time_zone`, `creme_core_cremeuser`.`theme`, `creme_core_cremeuser`.`language`, `creme_core_cremeuser`.`json_settings` FROM `creme_core_cremeuser` WHERE `creme_core_cremeuser`.`id` = 82 LIMIT 21
4. INSERT INTO `creme_core_historyline` (`entity_id`, `entity_ctype_id`, `entity_owner_id`, `username`, `date`, `type`, `value`) VALUES (1258, 73, 82, '', '2024-01-31 15:25:31.233056', 2, '[\"Hayao Miyazaki\", [\"image_id\", 1259]]') RETURNING `creme_core_historyline`.`id`
5. SELECT `creme_core_historyconfigitem`.`relation_type_id` FROM `creme_core_historyconfigitem`
6. SELECT `assistants_alert`.`id`, `assistants_alert`.`user_id`, `assistants_alert`.`title`, `assistants_alert`.`description`, `assistants_alert`.`is_validated`, `assistants_alert`.`trigger_date`, `assistants_alert`.`trigger_offset`, `assistants_alert`.`reminded`, `assistants_alert`.`entity_content_type_id`, `assistants_alert`.`entity_id` FROM `assistants_alert` WHERE (`assistants_alert`.`entity_id` = 1258 AND `assistants_alert`.`is_validated` = 0 AND JSON_CONTAINS_PATH(`assistants_alert`.`trigger_offset`, 'one', '$.\"cell\"'))
```

Reply:
I presume the query is caused by `creme.assistants.signals._update_alert_trigger_date()` ; it's totally OK. But it means that this test case with fail when the app "assistants" is not installed. It's not perfect but I prefer we keep it as is ; could you just just add a comment about this issue please.

Reply:
ok, I added a comment, thanks for the explanation !